src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidTestCommand.cs

```diff
@@ -87,11 +94,28 @@ protected override Task<ExitCode> InvokeInternal()
                     runner.KillApk(apkPackageName);
                 }
 
-                // No class name = default Instrumentation
-                runner.RunApkInstrumentation(apkPackageName, _arguments.InstrumentationName, _arguments.InstrumentationArguments, _arguments.Timeout);
-
                 using (_log.BeginScope("Post-test copy and cleanup"))
                 {
+                    // No class name = default Instrumentation
```

Comment:
this logging scope moving was a weird mouse dragging error, will undo before merge.